kuma/core/tests/test_locale_middleware.py

```diff
@@ -63,7 +71,13 @@ def test_locale_middleware_picker(accept_language, locale, client, db):
 def test_locale_middleware_fixer(original, fixed, client, db):
     '''The LocaleStandardizerMiddleware redirects non-standard locale URLs.'''
     response = client.get('/%s/' % original)
-    assert response.status_code == 302
+    if original == '':
```

Comment:
So we are testing the URL `//`? WEIRD

Reply:
😄 good catch! I added `('/%s/' % original) if original else '/'` to get rid of that weirdness.